lib/sisimai/rfc3834.rb

```diff
@@ -15,8 +15,8 @@ class << self
             |Out[ ]of[ ]Office:
           )
         /xi,
-      }
-      Re1 = { :endof  => %r/\A__END_OF_EMAIL_MESSAGE__\z/ }
+      }.freeze
+      Re1 = { :endof  => %r/\A__END_OF_EMAIL_MESSAGE__\z/ }.freeze
```

Comment:
__[rubocop]__
* Layout/SpaceAroundOperators: Operator `=>` should be surrounded by a single space. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

<!-- Comment by [SideCI](https://sideci.com) -->